package.json

```diff
@@ -218,6 +219,7 @@
     "loadjs": "^3.5.4",
     "lodash-es": "^4.17.11",
     "loop-breaker": "^0.1.0",
+    "lru-cache": "^5.1.1",
```

Comment:
Used to cache `CodeMirror.Doc` instances. Basically I want to hold on to the last few, so that the undo buffer is retained if you're switching back and forth between a couple projects, but don't want to build up an indefinitely growing collection of them.

Reply:
Yes, makes sense